tools/sync_test_server/ros-testing-server.js

```diff
@@ -112,7 +114,8 @@ function doStartRealmObjectServer(onSuccess, onError) {
                 winston.info(`${data}`);
             });
 
-            waitForRosToInitialize(100, onSuccess, onError, Date.now());
+            // The interval between every health check is 0.5 second. Give the ROS 15 seconds to get fully initialized.
+            waitForRosToInitialize(30, onSuccess, onError, Date.now());
```

Comment:
make sure this works for both docker images, I had longer startup times when using `docker-02` if I remember (with the Shield)